llvm/include/llvm/IR/User.h

```diff
@@ -54,7 +54,10 @@ class User : public Value {
   void *operator new(size_t Size) = delete;
 
   /// Indicates this User has operands "hung off" in another allocation.
-  struct HungOffOperandsAllocMarker {};
+  struct HungOffOperandsAllocMarker {
+    /// The number of operands for this User.
+    const unsigned NumOps;
```

Comment:
```suggestion
    unsigned NumOps;
```
Avoid (non-static) const members.